tracer/src/Datadog.Trace/Agent/AgentWriter.cs

```diff
@@ -333,12 +336,15 @@ async Task InternalBufferFlush()
 
                         var success = await _api.SendTracesAsync(buffer.Data, buffer.TraceCount, CanComputeStats, droppedP0Traces, droppedP0Spans).ConfigureAwait(false);
 
+                        TelemetryMetrics.Instance.Record(Count.TraceSent, buffer.TraceCount);
```

Comment:
:oh: I missed that we don't have a SpanSent metric. Why? Because we can compute it from Created, dropped and sampled?

Reply:
Basically, yeah.